src/requests.rs

```diff
@@ -421,7 +393,11 @@ request!(RoomMessageRequest from OriginalRoomMessageRequest extracts room_id: st
 request!(KeysBackupRequest from OriginalKeysBackupRequest extracts version: string and groups rooms);
 
 // Specific conversion for signature upload as they can be returned directly by
-// some verification API and not via outgoing requests.
+// some verification API and not via outgoing requests. If returned by outgoing
+// requests they would need to be marked as sent using `id`, otherwise no need
```

Comment:
```suggestion
// some verification API and not via `outgoing_requests()`. If returned by 
// `outgoing_requests()` they would need to be marked as sent using `id`, otherwise no need
```